arangod/GeneralServer/CommTask.h

```diff
@@ -123,9 +123,7 @@ class CommTask : public std::enable_shared_from_this<CommTask> {
 
   /// @brief send response including error response body
   void addErrorResponse(rest::ResponseCode, rest::ContentType,
-                        uint64_t messageId, int errorNum, std::string const&);
-  void addErrorResponse(rest::ResponseCode, rest::ContentType,
-                        uint64_t messageId, int errorNum);
+                        uint64_t messageId, int errorNum, char const* errorMessage = nullptr);
```

Comment:
This is the perfect use case for a `string_view`. It can be implicitly constructed (one can argue about that) from `string` and `const char*` and you can extract strings from a `Slice` as view. When explicit ownership is not required it is the most versatile solution.